models/uniswap/ethereum/uniswap_ethereum_pools.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    schema = 'uniswap_ethereum',
+    alias = 'pools',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['pool'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "uniswap",
+                                \'["hildobby"]\') }}'
+    )
+}}
+
+SELECT 'ethereum' AS blockchain
+, 'uniswap' AS project
+, 'v2' AS version
+, pool
```

Comment:
Currently there is no `pool` and `fee` fields for `uniswap_v2_ethereum. Factory_evt_PairCreated`.
Maybe you mean `pair` for the pool?
```suggestion
, pair as pool
```
https://dune.com/queries?d=1&category=decoded_project&namespace=uniswap_v2&contract=Factory&blockchains=ethereum&abi=PairCreated